slave/cracker.go

```diff
@@ -0,0 +1,31 @@
+package slave
+
+import (
+	b64 "encoding/base64"
+	"fmt"
+
+	"github.com/aaronang/cong-the-ripper/lib"
+	"github.com/aaronang/cong-the-ripper/slave/brutedict"
+	"github.com/aaronang/cong-the-ripper/slave/hasher"
+)
+
+func Execute(task lib.Task, slave *Slave) {
+	bd := brutedict.New(&task)
+	digest, _ := b64.StdEncoding.DecodeString(task.Digest)
```

Comment:
You cannot compare `[]byte("WTpSrbQAR8IMSK9uMoOQEXfKy+2FojN8yEz+T1n21uE=")` with the `key []byte`, because that is not the same thing. You have to decode the string `WTpSrbQAR8IMSK9uMoOQEXfKy+2FojN8yEz+T1n21uE=` to `[]byte`, or encode the `key` to a string. I chose the former, because then the cracker loop does not need to do the encoding every time.